dev-python/google-auth-oauthlib/google-auth-oauthlib-0.2.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
```

Comment:
Did/could you test with python3.7?

Reply:
The lib and my script using it both work fine in a 3.7 venv.

In tree, `dev-python/google-auth` and `dev-python/requests-oauthlib` do not have `python3_7` flag. Do you want me to bump this flag on all deps and this new lib now, or in a separate PR ?

Reply:
I'd say better to do it in a separate PR, not to make this too complicated. Plus it gives you time to test it's working properly. When added later, you can just modify the PYTHON_COMPAT and no revbump needed, unless you do other changes in the ebuild. So it's quite simple to add python3.7 later.